src/World.cpp

```diff
@@ -3709,6 +3709,35 @@ bool cWorld::HasEntity(UInt32 a_UniqueID)
 
 
 
+OwnedEntity cWorld::RemoveEntity(cEntity & a_Entity)
+{
+	// Check if the entity is in the chunkmap:
+	auto Entity = m_ChunkMap->RemoveEntity(a_Entity);
+
+	// If not, check if the entity is in the queue to be added to the world:
+	if (Entity == nullptr)
```

Comment:
To save on indentation here, would checking for `!= nullptr` and returning in that case not be better?

Reply:
Changed